packages/langium/src/services.ts

```diff
@@ -154,6 +155,7 @@ export type LangiumDefaultSharedServices = {
         DocumentLinkProvider?: DocumentLinkProvider
         ExecuteCommandHandler?: ExecuteCommandHandler
         InlayHintProvider?: InlayHintProvider
+        WorkspaceSymbolProvider?: WorkspaceSymbolProvider
```

Comment:
We don't have to make this optional if we have a good default?

Reply:
We currently make every LSP service optional, so adopters can just remove them if they want to.